packages/react-composites/src/composites/CallComposite/components/CallArrangement.tsx

```diff
@@ -81,6 +81,9 @@ export const CallArrangement = (props: CallArrangementProps): JSX.Element => {
   /* @conditional-compile-remove(one-to-n-calling) */
   const { callStatus } = useSelector(callStatusSelector);
 
+  /* @conditional-compile-remove(PSTN-calls) */ /* @conditional-compile-remove(one-to-n-calling) */
+  const disableControlsForHoldScreen = adapter.getState().page === 'hold';
```

Comment:
can this be done at the selector level instead of plumbing through the composite? i.e. the usePropsFor(CameraButton) will return a `disabled: true` when on hold